terraform/modules/sft/srv-announcer-iam.tf

```diff
@@ -0,0 +1,60 @@
+resource "aws_iam_user" "srv-announcer" {
+  name          = "${var.environment}-srv-announcer"
+  force_destroy = true # TODO: Add a comment explaining this. Does this mean
```

Comment:
has this been addressed?

Reply:
I copied this from [here](https://github.com/zinfra/cailleach/blob/5701e7412d2fc07fcd3b5b1a65b0dace09aacf2c/environments/exxon-dev/terraform/sft/resources.srv-announcer.tf#L10). @lucendio Do you remember why you added this? It looks dangerous, in the sense that running terraform could make the SRV announcer stop working.

Reply:
Actually, nevermind. It doesn't mean what I thought it meant, from [terraform docs](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/iam_user#force_destroy):

> When destroying this user, destroy even if it has non-Terraform-managed IAM access keys, login profile or MFA devices. > Without force_destroy a user with non-Terraform-managed access keys and login profile will fail to be destroyed.

Now I am thinking maybe we shouldn't have this set to `true`. WDYT @lucendio?